Vagrantfile

```diff
@@ -166,28 +166,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define "debian8-agent", autostart: false do |agent|
     agent.vm.box = "debian/jessie64"
+    agent.vm.box_version = '8.11.0'
     agent.vm.hostname = 'debian8-agent.example.com'
     agent.vm.network  :private_network, ip: "192.168.52.17"
     agent.vm.provision :shell, :path => "tests/provision_basic_debian.sh"
     agent.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-agent.pp"
     agent.vm.provision :shell, :inline => "facter --custom-dir=/vagrant/lib/facter sensu_agent"
   end
 
-  config.vm.define "win2008r2-agent", autostart: false do |agent|
```

Comment:
getting rid of this platform?

Reply:
We can still support it just not in Vagrant, I could not get it to work. Once I got rid of issues with older powershell not working with vbguest, I ran into issues with older powershell not being able to download Puppet MSI or Chocolately installer due to not supporting modern TLS that all websites are now using.  The only only solution I could think of is to create a custom box of Windows 2008R2 that has newer powershell installed and just doesn't seem worth the effort when it doesn't affect this module's ability to support a platform.

Also I'm pretty sure Windows 2008R2 is EOL.  I can open a different pull request to remove support for that platform, but figured could leave as Sensu Go lists it as supported.

Reply:
https://www.microsoft.com/en-us/cloud-platform/windows-server-2008

EOL back in January.

Reply:
Agreed, lets just remove it from Vagrant. If it turns into a problem supporting it later, we can re-evaluate then.